client/public/locales/en/common.json

```diff
@@ -1,263 +1,265 @@
 {
-    "actions": {
```

Comment:
See if you can re-save this file without messing up the entire formatting

Reply:
Just pushed, inconveniently I had to do it with nano because VSCode insists on formatting on save.

Reply:
I format using prettier in VSCode and it works fine for me. Is it using the prettier config in the client directory properly?